sympy/plotting/pygletplot/color_scheme.py

```diff
@@ -66,7 +67,7 @@ def __init__(self, *args, **kwargs):
             raise ValueError("Color gradient not properly initialized. "
                              "(Not a ColorGradient instance.)")
 
-    def _interpret_args(self, args, kwargs):
+    def _interpret_args(self, *args, **kwargs):
```

Comment:
The method doesn't use `kwargs` at all, so maybe it could be removed from the parameters?